tests/core/utilities/test_encoding.py

```diff
@@ -1,7 +1,8 @@
 # encoding: utf-8
 
-from __future__ import unicode_literals
+# from __future__ import unicode_literals
 
+from unittest.mock import Mock
 import pytest
 import sys
```

Comment:
This is the nitpickiest of all, but I've been alphabetizing the imports whenever I touch them. (only inside the [PEP8 subgroups](https://www.python.org/dev/peps/pep-0008/#imports), so `unittest` would go after `sys` here).